src/main/routes/home.ts

```diff
@@ -77,6 +80,8 @@ async function caseHandler(req: AppRequest, res: Response) {
     caseSearchPage: true,
     caseActivities: req.session?.caseActivities,
     caseSearches: req.session?.caseSearches,
+    jurisdictions: app.locals.container.cradle.autoSuggestService.getJurisdictionsData(),
+    caseTypes: app.locals.container.cradle.autoSuggestService.getCaseTypesData(),
```

Comment:
see the above comment about putting the loading of data into a module. 

the example given on setting the data

```res.locals.CaseType = data```

is available throughout the application and nunjucks this means that there is no need to do the following

```
jurisdictions: app.locals.container.cradle.autoSuggestService.getJurisdictionsData(),
caseTypes: app.locals.container.cradle.autoSuggestService.getCaseTypesData(),
```